api/src/services/observation-service.ts

```diff
@@ -27,12 +27,11 @@ import { DBService } from './db-service';
 const defaultLog = getLogger('services/observation-service');
 
 const observationCSVColumnValidator: IXLSXCSVValidator = {
-  columnNames: ['SPECIES', 'COUNT', 'DATE', 'TIME', 'LATITUDE', 'LONGITUDE'],
-  columnTypes: ['number', 'number', 'date', 'string', 'number', 'number'],
+  columnNames: ['ITIS_TSN', 'ITIS_SCIENTIFIC_NAME', 'COUNT', 'DATE', 'TIME', 'LATITUDE', 'LONGITUDE'],
+  columnTypes: ['number', 'string', 'number', 'date', 'string', 'number', 'number'],
   columnAliases: {
     LATITUDE: ['LAT'],
-    LONGITUDE: ['LON', 'LONG', 'LNG'],
-    SPECIES: ['TAXON']
```

Comment:
Given we now have two columns for tracking species, we should decide on new column aliases for `ITIS_TSN` and `ITIS_SCIENTIFIC_NAME`.

Reply:
As discussed, we will remove ITIS_SCIENTIFIC_NAME, and support the following aliases for ITIS_TSN:
```
species
taxon
itis
itis_tsn
```